website/docs/r/user_data.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
can we add this to the sidebar? :)

Reply:
thanks for pointing out the sidebar! I've updated the PR!